eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/service/config/instanceoperation/ConnectorConfigController.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.console.service.config.instanceoperation;
+
+
+import org.apache.eventmesh.dashboard.console.entity.config.ConfigEntity;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class ConnectorConfigController {
+
+    public List<ConfigEntity> getConnectorConfigFromInstance(Long clusterId, Long id) {
+        return null;
+    }
+}
```

Comment:
Do you want to RPC a specific connector instance for its configuration in this class?

A `Controller` class annotated with `@Service` seems incongruous.

Reply:
> RPC a specific connector instance

which means initiating a web request from dashboard to service instance.

I have not been able to gain an understanding of the function of this class through its code.

Reply:
this class just want to get a Series of Config message from a instance.I don't know the specific implementation

Reply:
This class should be placed in `service` module. Or you may leave a javadoc and do it latter.

Reply:
> This class should be placed in module. Or you may leave a javadoc and do it latter.service
ok，I choose leave a javadoc，are the other classes in the same category as it?

> Also, EventMesh doesn't support to "get" configs from a running connector instance. Only Runtime instance supports this feature.
ok，I know.Let's keep it for now.

Reply:
Let's keep these classes for now and leave javadocs to record their usage and migration plan.